lib/namespaced_authorizing_processor.rb

```diff
@@ -0,0 +1,244 @@
+# frozen_string_literal: true
+
+# The JSONAPI Authorizing processor for namespaced policies.
+# rubocop:disable Metrics/ClassLength, Rails/DynamicFindBy
+class NamespacedAuthorizingProcessor < JSONAPI::Authorization::AuthorizingProcessor
```

Comment:
So I've had to create a processor for our Pundit policies that are namespaced.  This helps us have different restriction policies for public versus protected resources.